api-docs/api-reference/methods/users/get-accessible-domains-for-user-v2.0.rst

```diff
@@ -0,0 +1,93 @@
+.. _get-accessible-domains-for-user-v2.0:
+
+Get accessible domains for user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/{userId}/RAX-AUTH/domains
+
+Use the get accessible domains for user operation to retrieve the list of
+domains a user has access to.
+
+Request
+-------
+
+This table shows the header and URI parameters for the request:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    X-Auth-Token, Header String *(Required)*, A valid authentication token.
+    {userId}, URI String *(Required)*, A user ID assigned by system when user is added.
+
+Response
+--------
+
+The following table shows the body parameters for the retrieve domains
+response:
+
+.. csv-table::
+    :header: Name, Type, Description
+    :widths: 2, 2, 2
+
+    RAX-AUTH:domains, Object *(Required)*, The collection of domains that the authenticated user has permission to view.
+    RAX-AUTH:domains.\ **RAX-AUTH:domains**,Object *(Required)*, An object that contains the domain configuration attribute settings.
+    RAX-AUTH:domain.\ **id**, String *(Required)*, The unique id for the domain.
+    RAX-AUTH:domain.\ **sessionInactivityTimeout**, String *(Required)*, Session inactivity timeout property used across all Rackspace UIs.
+    RAX-AUTH:domain.\ **enabled**, Boolean *(Optional)*, Indicates whether the domain is enabled.
+    RAX-AUTH:domain.\ **description**, String *(Optional)*, The domain description.
+    RAX-AUTH:domain.\ **name**, String *(Optional)*, The domain name.
+    RAX-AUTH:domain.\ **rackspaceCustomerNumber**, String *(Required)*, The Rackspace customer number.
+    RAX-AUTH:domain.\ **domainMultiFactorEnforcementLevel**, String *(Optional)*, "If present, this extended attribute specifies the multi-factor authentication enforcement policy that applies to accounts within the specified domain.
+
+    ``REQUIRED`` - Users within the domain must use multi-factor authentication to access their account.
+
+    ``OPTIONAL`` - Users have the option to authenticate using multi-factor authentication."
+
+**Example: Get accessible domains for user: XML response**
+
+.. code::
+
+    < HTTP/1.1 200 OK
```

Comment:
When I was working with Margaret, she told me to always include the headers on example responses and to include a table if non-standard headers where returned and needed a description to explain the header.